### ISSUE TITLE
Any Plans to have Flow Bakery Startup app in VoK

### ISSUE THREAD

**Issue by NiranjanShah** [mvysny/vaadin-on-kotlin#15]

**Comment by mvysny** [comment]
I wanted to convert Bakery to Kotlin; unfortunately I can't provide Bakery sources on Github since it's delivered only after registration to Vaadin. Maybe later Vaadin will provide Kotlin Bakery via the website as well, but there are currently no plans regarding that unfortunately.

**Comment by mvysny** [comment]
Unfortunately Vaadin policy of downloading Flow Bakery on demand prevents me to publish Bakery sources on Github for free access.

Porting Flow Bakery to VoK is however a good exercise and clearly demonstrates how much simpler VoK is, as opposed to Spring or JavaEE stack.

Closing as won't fix; please reopen if the original Bakery becomes available freely on Github.
